client/dist/styles.css

```diff
@@ -203,12 +203,97 @@ main {
 
 /**********************************************/
 /****** Style for Overview widget****/
-#expanded-image {
-  width: 90vw;
+
+.overview-container {
+  margin: 100px;
+}
+
+/* image gallery */
+
+.default-image-gallery {
+  padding-bottom: 100px;
+  margin-bottom: 100px;
+  height: 500px;
+}
+
+.default-view {
+  position: relative;
+  top: 0;
+  left: 0;
+}
+
+ .default-image {
+  height: 50vh;
+  width: 50vw;
+  position: absolute;
+  cursor: zoom-in;
+}
+
+
+.thumbnail {
+  display: flex;
+  /* flex: 1 1 0; */
+  height: 50px;
+  width: 50px;
+  border: 1px solid black;
+  box-sizing: border-box;
+
+}
+
+.selected-thumbnail {
+  display: flex;
+  /* flex: 1 1 0; */
+  height: 50px;
+  width: 50px;
+  border: 2px solid rgb(125, 125, 188);
+  box-sizing: border-box;
+
+}
+
+.thumbnail-container {
+  position: relative;
+  display: flex;
+  gap: 7px;
```

Comment:
Favor rem or em units. Read more about ["Relative length units"](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units) (scroll down to that heading) https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units